firmware/temperature_detection/volt_to_temp.ino

```diff
@@ -0,0 +1,23 @@
+//for ztp-115m
+void setup() {
+  // put your setup code here, to run once:
+  Serial.begin(115200);
+  Serial.println("hello");
+}
+void loop() {
+  // put your main code here, to run repeatedly:
+  // read the input on analog ADC1_0:
+  int sensorValue = analogRead(36);
+  Serial.println("Sensor Value");
+  Serial.println(sensorValue);
+  float voltage;
+  voltage = 3.3 * (sensorValue/4095.0);
+  Serial.println("Voltage");
+  Serial.println(voltage);
+  float temperature;
+  temperature = 8.294633642930862*10^(-5)*voltage^(2) + .0088*voltage + 1.2603;
```

Comment:
```suggestion
  temperature = (voltage * (106.093 + voltage) + 15194.2) / 12055.99;
```
The original formula doesn't consider the poor precision of AVR floating point operations, and uses the `^` operator to represent powers, but C++ interprets it as an [exclusive or](https://en.wikipedia.org/wiki/Exclusive_or). 

My simplified formula should give us a maximum (theoretical) error of `3e-6` (0.000003), so it looks good enough.